lib/internal/process/nextTick.js

```diff
@@ -0,0 +1,228 @@
+'use strict';
+
+exports.setup = setupNextTick;
+
+function setupNextTick(NativeModule) {
```

Comment:
what do you mean? `require` and `NativeModule.require` is literally the same thing in built-in modules: https://github.com/nodejs/node/blob/master/src/node.js#L1004